### DISCUSSION TITLE
Module: Default Credential Scanning

### DISCUSSION THREAD

**Discussion by aconite33** [blacklanternsecurity/bbot#727] [Ideas]
Add a module that will search for default credentials within an environment. Should operate after TCP ports have been enumerated.

Examples Projects:

* https://github.com/projectdiscovery/nuclei
* https://github.com/lanjelot/patator
* https://github.com/nmap/nmap

**Comment by liquidsec** [comment]
This is pretty well covered by nuclei's default-login templates. Is there something beyond what nuclei can provide that you had in mind?

**Comment by aconite33** [comment]
The goal is to replicate what we do with https://github.com/blacklanternsecurity/zmap-asset-inventory in order to achieve similar coverage.

SSH
Telnet
FTP / Anonymous FTP
NFS
iSCSI

All Non HTTP/HTTPS protocols

**Comment by Sh4d0wHunt3rX** [comment]
I was looking for default credentials scanning on hacktricks, was wondering if this will be added into BBOT in future or it's not in roadmap?

![image](https://github.com/blacklanternsecurity/bbot/assets/15929497/ef41908b-5f2d-4779-aafa-42481be9ba04)
